prisonstaffhub-specs/src/test/groovy/uk/gov/justice/digital/hmpps/prisonstaffhub/pages/ActivityPage.groovy

```diff
@@ -21,11 +21,11 @@ class ActivityPage extends DatePickerPage {
         printButton { $('#printButton') }
         tableRows { $('table.row-gutters tr') } // Avoid the calendar table rows
         bodyRows { $('table.row-gutters tr', 1..-1) }
-        locations { bodyRows*.$('td', 1)*.text() }
-        nomsIds { bodyRows*.$('td', 2)*.text()  }
-        flags { bodyRows*.$('td', 3)*.$('span') }
-        events { bodyRows*.$('td', 4)*.text() }
-        eventsElsewhere { bodyRows*.$('td', 5).collect { it.$('ul li')*.text() } }
+        locations { bodyRows*.$('td', 2)*.text() }
```

Comment:
A bit confused as to why you've had to change this? You've not added any new columns, have you?

Reply:
Ah, you've had to remove the info column on print.

Reply:
Yeah I have.  Because I'm updating the state to handle the appropriate print version I haven't tweaked the content of the existing table in most examples so as not to disrupt the visual on screen.  For the offender first initial and redacted offender number for example i've added hidden columns in so that they can be printed and the ones on screen aren't affected.